docs/guides/02_page_layer_widgets.md

```diff
@@ -74,16 +74,16 @@ In this case, we're going to create a new source that can feed a layers & widget
 ✔ Type a query: select store_id, storetype, revenue, address, the_geom_webmercator from retail_stores
 ```
 
-After filling all the requirements, it creates a new file `src/data/sources/StoresSource.js` that will contains the following basic structure:
+After filling all the requirements, it creates a new file `src/data/sources/storesSource.js` that will contains the following basic structure:
 
 ```javascript
-export const STORES_SOURCE_ID = 'StoresSource';
+const STORES_SOURCE_ID = 'StoresSource';
 
-export const storesSource = {
+export default {
   id: STORES_SOURCE_ID,
   data: `
-		select store_id, storetype, revenue, address, the_geom_webmercator from retail_stores
-	`,
+    select store_id, storetype, revenue, address, the_geom_webmercator from retail_stores
```

Comment:
Please add `cartodb_id` to this source (and kpiSource too). It is required for the filtering to work nicely